.gitignore

```diff
@@ -4,6 +4,7 @@
 # Don't ignore git settings
 !*.gitignore
 !*.gitattributes
+!*.github/
```

Comment:
Isn't this redundant? Since only `*.md` files are in this directory which are un-ignored at new (old) line 10 (9)?